client/xr/hand_tracker.cpp

```diff
@@ -77,6 +77,10 @@ std::optional<std::array<xr::hand_tracker::joint, XR_HAND_JOINT_COUNT_EXT>> xr::
 	if (!locations.isActive)
 		return std::nullopt;
 
+	// bail if none of the joints are valid/tracked
+	if (!std::ranges::any_of(joints_pos, [](const auto & loc) { return loc.locationFlags != 0; }))
+		return std::nullopt;
+
```

Comment:
Is the headset returning something with `isActive = true` and invalid joints?

Per https://registry.khronos.org/OpenXR/specs/1.1/html/xrspec.html#XrHandJointLocationsEXT
> If the returned isActive is true, the runtime must return all joint locations with both XR_SPACE_LOCATION_POSITION_VALID_BIT and XR_SPACE_LOCATION_ORIENTATION_VALID_BIT set. Although, in this case, some joint space locations may be untracked (i.e. XR_SPACE_LOCATION_POSITION_TRACKED_BIT or XR_SPACE_LOCATION_ORIENTATION_TRACKED_BIT is unset).